kubernetes/report-ui/templates/refelx-grafana-ingress.yaml

```
@@ -0,0 +1,136 @@
+        add_header Set-Cookie "ref_grafana_hostgroup=$hostgroup; Path=/; HttpOnly; Secure; Samesite=Lax; Max-Age=86400";
+        add_header Set-Cookie "ref_grafana_orgId=$orgId; Path=/; HttpOnly; Secure; Samesite=Lax; Max-Age=86400";
+        add_header Set-Cookie "ref_grafana_report=$report; Path=/; HttpOnly; Secure; Samesite=Lax; Max-Age=86400";
+        #ref_grafana_session already set from grafana ingress
+        set $redirect_url "$scheme://$host{{ $basehref }}/report/grafana/$report/";
+
+        return 302 $redirect_url;
+      }
+    nginx.ingress.kubernetes.io/force-ssl-redirect: "true"
+  name: grafana-redirect-report-ingress
+spec:
+  ingressClassName: {{ .Release.Name }}
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)